Src/Test.UnitTests.Sarif.PatternMatcher/SearchSkimmerTests.cs

```diff
@@ -487,6 +492,115 @@ public void SearchSkimmer_ShouldNotEvaluateTooLargeFiles()
             }
         }
 
+        [Fact]
+        public void SearchSkimmer_ShouldEvaluateFilesUnderLimit()
+        {
+            // `MaxFileSizeInKilobytes` values to test.
+            int[] testCases = new int[]
+            {
+                (int)uint.MinValue + 1,
+                10000,
+                100000,
+                1000000,
+                1024 * 10000 - 1,
```

Comment:
Is it possible to make the default a constant shared between the product (tool) and tests. minimally a const within the tests since it is used in multiple cases.

Reply:
Added a constant to the test to cover this.